input/httplisten/httplisten.go

```diff
@@ -85,7 +89,43 @@ func (i *InputConfig) Start(ctx context.Context, msgChan chan<- logevent.LogEven
 	go func() {
 		logger.Infof("accepting POST requests to %s%s", i.Address, i.Path)
 		if i.ServerCert != "" && i.ServerKey != "" {
-			err = http.ListenAndServeTLS(i.Address, i.ServerCert, i.ServerKey, nil)
+			var tlsConfig *tls.Config
+			srvCert, err := tls.LoadX509KeyPair(i.ServerCert, i.ServerKey)
+			if err != nil {
+				logger.Fatal(err)
+				return
+			}
+
+			if i.CA != "" {
+				// enable client certificate
+				f, err := os.Open(i.CA)
+				if err != nil {
+					logger.Fatal(err)
+					return
+				}
+				content, err := ioutil.ReadAll(f)
+				f.Close()
```

Comment:
at least log error?

Reply:
error is logged below - line 109

Reply:
I mean `f.Close()` will return error, but you ignored.